torch/csrc/jit/ir/ir.cpp

```diff
@@ -1079,6 +1079,11 @@ bool Node::hasSideEffects() const {
     case prim::rpc_sync: // It represents RPC message sent.
     case prim::rpc_remote: // It represents RPC message sent.
     case aten::wait: // It can represent RPC message received.
+    #ifdef __HIP_PLATFORM_HCC__
```

Comment:
You want to hide it from RocM, aren't you?
```suggestion
    #ifndef __HIP_PLATFORM_HCC__
```